### ISSUE TITLE
Site won't load at school

### ISSUE THREAD

**Issue by BillMacInc** [photopea/photopea#1939]
I would like to use this program with my students....thought the http**s** would have no issue loading at our high school. What would cause the board to block it? Is there any thing i can do to allow to load.  Such a great program.

![image](https://user-images.githubusercontent.com/61197842/74749882-d695a900-5238-11ea-8b1c-b22f15dafab6.png)

**Comment by photopea** [comment]
Hi, that is really strange.

Can you try it in Incognito mode? Can you try it e.g. on your phone on WiFi?

**Comment by BillMacInc** [comment]
I will try Incognito brb.  Phone it too small...but it won't work
anyway...i tried on m laptop on wifi

**Comment by BillMacInc** [comment]
No luck, We have photo shop element 14 installed, but i wanted them to have a way to work at home, or use the skills at home after the course is completed. Not all will buy the program we have, but your price is so reasonable (and even in free mode.....ads are only on the small right pane). Too bad it won't load, take so long to have IT here fix the problem..... :(

**Comment by photopea** [comment]
Could you tell me, where are you located? Do you have Optimum Online internet?

Could you go to Start, type "cmd", Enter (to open a terminal) and run commands "tracert photopea.com" + Enter? What do you see?

**Comment by BillMacInc** [comment]
The school has a lot of policies that prevent cmd line etc....but we have
super hi speed. Had no problem loading pixlr. But i think your app is ten
times better. We are in Ontario Canada.

**Comment by photopea** [comment]
I would need somebody from your side to assist me, by running cmd commands, etc.

First, you should ask the school authorities, if Photopea is blacklisted or blocked otherwise.

**Comment by photopea** [comment]
You can not really call it an internet access, if some websites are blocked, while others are "encouraged". It is more like a censored internet in China.

Maybe I could tweet to your school, so that they look into it?

**Comment by BillMacInc** [comment]
Yes, but the whole point was to just use a web site and not bother with
them, ...i wanted to use it in this semester...they will take for ever to
get back and solve the problem.

Do you think the ads could be the problem? I don't think it is blacklisted,
we use wix for web sites and it works no problem.....and other
online sites....

**Comment by BillMacInc** [comment]
I will try the cmd on my laptop...and see what is say...brb

**Comment by photopea** [comment]
I changed our DNS servers. Does it work now?

**Comment by BillMacInc** [comment]
I can't express to you how much I appreciate your hard work (both the web app and especially your support). I won't be able to test it until this Monday. I hope it works, I 'll let you know and if you need me to run any cmd codes, I'll do that too. 

Just for the people reading this.....I have tested every WEB photo editor out there (days and days....lol) and not one even comes close to Photopea. Speed wise, amount of features and how much time he spends trying to help ppl here...i read them all.

**Comment by BillMacInc** [comment]
I am still not able to log in on my laptop at work (wifi) and the student computes (they are on a network).

**Comment by photopea** [comment]
What do you mean by logging in?

We were discussing, if www.Photopea.com can be loaded in a web browser at your school.

**Comment by BillMacInc** [comment]
Logging was the wrong word.....I can't load the web page using several browsers. Same error page . At the top of this thread, i have an image.

**Comment by photopea** [comment]
Hi, I discussed it with my hosting provider. It is probably because of a network policy inside your network.

Could you tell me your public IPv4 address? You can see it here (when you open it from a school) https://www.whatismyip.com/

**Comment by BillMacInc** [comment]
24.51.244.63

**Comment by photopea** [comment]
My provider tells me that your IP address is not blocked by our servers. You should ask the administrator of your network about this issue.

**Comment by BillMacInc** [comment]
Ok, i appreciate your time. It will take a long time for them to solve the problem...but your site is worth it.

**Comment by BillMacInc** [comment]
Can you send me a private message....i need to ask you something but i don't want to have it public?? Do you have access to the email that i am using?

**Comment by photopea** [comment]
write it to support@photopea.com

**Comment by BillMacInc** [comment]
Ty
